### PR TITLE
Wizard page wifi field redo

### PR DESCRIPTION
**Pull request by marcelinkaaa**
added new darker icon and changed the text colors

### CODE REVIEW

**Comment by gubatron** [comment]
Thank you!
thanks-bow

File: android/src/com/frostwire/android/gui/views/GeneralWizardPage.java [hunk 1]
```diff
@@ -86,16 +86,20 @@ protected void onFinishInflate() {
...
         View.inflate(getContext(), R.layout.view_general_wizard_page, this);
+        final TextView textWifiOnly = (TextView) findViewById(R.id.view_general_wizard_page_wifi_only_text);
 
         checkSeedFinishedTorrents = (CheckBox) findViewById(R.id.view_general_wizard_page_check_seed_finished_torrents);
         checkSeedFinishedTorrents.setOnCheckedChangeListener(new OnCheckedChangeListener() {
             public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
                 checkSeedFinishedTorrentsWifiOnly.setEnabled(isChecked);
-                checkSeedFinishedTorrentsWifiOnly.setTextColor((isChecked) ? Color.WHITE : getContext().getResources().getColor(R.color.frostwire_gray_explanation_text));
+                checkSeedFinishedTorrentsWifiOnly.setTextColor((isChecked) ? Color.WHITE : getContext().getResources().getColor(R.color.frostwire_gray_explanation_text_dark));
+                textWifiOnly.setTextColor(checkSeedFinishedTorrents.isChecked() ? getContext().getResources().getColor(R.color.frostwire_gray_explanation_text) : getContext().getResources().getColor(R.color.frostwire_gray_explanation_text_dark));
```
Comment (gubatron) [inline android/src/com/frostwire/android/gui/views/GeneralWizardPage.java]: it's better if you write it like this (great try though)

```java
textWifiOnly.setTextColor(getContext().getResources().getColor(checkSeedFinishedTorrents.isChecked() ? R.color.frostwire_gray_explanation_text : R.color.frostwire_gray_explanation_text_dark));
```

Comment (gubatron) [inline android/src/com/frostwire/android/gui/views/GeneralWizardPage.java]: what changes is the color, no need to repeat the getColor() code.

Comment (gubatron) [inline android/src/com/frostwire/android/gui/views/GeneralWizardPage.java]: also, to make it nicer, less code repetition, we can assign the output of `getContext().getResources().getColor(...)` to a local variable `r`, and then just substitute for `r.getColor(...)` everywhere below.

Comment (marcelinkaaa) [inline android/src/com/frostwire/android/gui/views/GeneralWizardPage.java]: Thanks, that makes a lot of sense actually. I changed it for now and will attempt to try your second suggestion.

Comment (gubatron) [inline android/src/com/frostwire/android/gui/views/GeneralWizardPage.java]: please send the commit, I'm ready to merge this, I just made the text view functional.

**Comment by gubatron** [comment]
merged separately.

**Pull request closed without merging**